app/src/main/kotlin/social/plasma/relay/Relays.kt

```diff
@@ -0,0 +1,24 @@
+package social.plasma.relay
+
+import kotlinx.coroutines.currentCoroutineContext
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+class Relays(
+    private val okHttpClient: OkHttpClient,
+) {
+
+    suspend fun subscribe(
+        url: String,
+    ): Flow<String> {
+        val messages = MutableSharedFlow<String>()
+        okHttpClient.newWebSocket(
```

Comment:
concerned about leaks here ... might want to eventually return a type that includes the websocket instance and can be cleaned up